src/Checkout.vue

```diff
@@ -60,23 +60,29 @@ export default {
       loadStripeCheckout(this.pk, 'v3', () => {
         try {
           let stripe = window.Stripe(this.pk);
-          stripe.redirectToCheckout({
-            billingAddressCollection: this.billingAddressCollection,
-            cancelUrl: this.cancelUrl,
-            clientReferenceId: this.clientReferenceId,
-            customerEmail: this.customerEmail,
-            items: this.items,
-            locale: this.locale,
-            sessionId: this.sessionId,
-            submitType: this.submitType,
-            successUrl: this.successUrl,
-          });
+          if (this.sessionId == null) {
+            stripe.redirectToCheckout({
+              billingAddressCollection: this.billingAddressCollection,
+              cancelUrl: this.cancelUrl,
+              clientReferenceId: this.clientReferenceId,
+              customerEmail: this.customerEmail,
+              items: this.items,
+              locale: this.locale,
+              sessionId: this.sessionId,
```

Comment:
Is this still necessary if the `sessionId` prop is null?

Reply:
Nope, good catch 😄

Reply:
@tbrown1979 kindly remove the `sessionId` key and update na PR. Thanks :)